sbt-domtypes/src/main/scala/com/raquo/sbt/domtypes/CustomElementsManifest.scala

```diff
@@ -0,0 +1,158 @@
+package com.raquo.sbt.domtypes
+
+import upickle.default.*
+
+// This is taken directly from the raquo/laminar-shoelace-components/project/CustomElementsManifest.scala file.
+// See: https://github.com/raquo/laminar-shoelace-components/blob/f2894a247b197534e7a30f57287bbf475fc9a242/project/CustomElementsManifest.scala
+// See also: See https://github.com/webcomponents/custom-elements-manifest
+final case class CustomElementsManifest(
+    schemaVersion: String, // "1.0.0"
+    readme: String, // ""
+    modules: List[CustomElementsManifest.Module]
+)
```

Comment:
I agree with moving the manifest parsing logic into SDT. It is free from Shoelace specifics (I think), and keeping it here will let other projects like https://github.com/elgca/laminar-mdui-components and https://github.com/cornerman/scala-web-components-codegen reuse this code without duplicating it.

Reply:
Good idea generally to move it here, but I am not sure, whether we are ready yet. I have tried it with shoelace and some other custom-elements.json. And I already needed to update it. Maybe it makes sense to stabilize it a bit more.
For now, i would probably prefer to have a local copy in my project to be more flexible when using it with other custom-elements in the wild.